edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -248,7 +248,16 @@ def _verify_jwt_signature(token, jwt_issuer, decode_symmetric_token):
         raise jwt.InvalidTokenError(exc_info[2]) from token_error
 
 
-def _verify_jwk_signature_using_keyset(token, key_set, jwt_issuer):
+def verify_jwk_signature_using_keyset(token, key_set, aud=None, iss=None, verify_signature=True, verify_exp=True):
+
```

Comment:
This should get a docstring if it's public. (PR should also include a changelog.)

Reply:
Added.